### ISSUE TITLE
Improving sprites in player plus metadata in player

### ISSUE THREAD

**Issue by tfchristie** [netmix/radio-station#429]
Hey Guys - 
**Describe the bug**
As you can see in the attached image file, I'm having trouble with both the nav sprite as well as spacing.  Wondering if you can let me know where the css file would be to modify look/feel of the player other than through the Settings windows.

In that same attached image, the Show Name is showing up but the current track is not.
Looking at the console (debug mode is on), I am seeing this:
`Now Playing:
(index):6382 {text: 'Jazz in Flight with Theresa Cuccia | Speak Low - Sonny Clark from Corrine Corinna', title: 'Jazz in Flight with Theresa Cuccia | Speak Low', artist: 'Sonny Clark from Corrine Corinna'}`
So the info is there, it just isn't displaying as expected. 
Can you let me know how idiotic I am to not see the obvious in settings somewhere?
Thanks so much!

p.s. this probably isn't a bug report issue but I wasn't sure where else to put the query.

**To Reproduce**
1. display the page

**Expected behavior**
To see a single + or - for volume control with some added spacing perhaps?  Also to see the currently playing track.

**Screenshots**
Attached.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - Device: Desktop
 - OS: Win 10
 - Browser: Chrome  version 106
 
![radioplayer](https://user-images.githubusercontent.com/4172639/193139729-7ca085de-1ac2-461e-8a14-b3d5e39c14d2.jpg)

**Comment by majick777** [comment]
@guydonges This could be a bug or a conflict with your theme. I've recently redone the player graphics, so I recommend trying out the latest development version to see if that fixes your display issues:
https://github.com/netmix/radio-station/tree/develop

Similarly, have been improving the now playing track display in the player bar and overall player bar responsiveness. If you enable "Beta testing" from your Account page, you can install the latest Pro development version also, and this might fix your existing track display issues.

After that, let us know if you are still having problems on either front or not.

**Comment by tfchristie** [comment]
Thanks, Tony -
I shall check it out on our test server and see what happens and report
back.

Thom

**Comment by guydonges** [comment]
Okay, current status:
I couldn't change the default background color for the player bar at all -
it remained transparent no matter what I put in there.  I finally went into
the css file and forced it to #000 for now.
The image attached shows current status of it.  It seems about 30 pixels
spaced above bottom of browser window.
And no metadata is coming through at all now (coming from an icecast server)
Still plays fine, though which is probably the most important thing.  Haa!

**Comment by tonyzeoli** [comment]
I caught that same transparent bug and am glad you were able to fix that
manually. It was the background setting for the player that wasn't saving
the selected background color. Tony has a fix for it anyway in the upcoming
release. We also have a fix for the track titles in the player. What
happened there was that there are two different methods a server can use to
extract that data and display it on the screen. Some servers have it
installed while others do not. We weren't aware of that until we dissected
another player plugin and saw that the player needed to support both
methods because you don't know which method is installed on the server.
That is now in our development version, which you can take from the free
repo at https://github.com/netmix/radio-station. You can switch the
drop-down from Master to develop and then download the code and try it.

Tony Zeoli

**Comment by tfchristie** [comment]
Yeah, Tony pointed me to the dev version and the beta of the pro version
which is what I installed on my development box.  I'm happy this is seeing
more progress.
Any update on adding "Reporters" to the mix of roles for our news shows?
Just had to ask since we were communicating anyway.  Haaa!
